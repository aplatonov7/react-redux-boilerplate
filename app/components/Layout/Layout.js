import React from 'react'
import s from './Layout.scss'

const Layout = () => (
  <main className={s.root}>
    Hello
  </main>
)

export default Layout