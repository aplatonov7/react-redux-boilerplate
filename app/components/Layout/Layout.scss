@import '../../styles/variables';

.root {
  color: #0ff;
}
