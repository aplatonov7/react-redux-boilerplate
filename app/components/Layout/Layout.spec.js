import React from 'react'
import renderer from 'react-test-renderer'
import Layout from './'

jest.mock('../Navigation', () => 'Navigation')

describe('Layout component', () => {
  const item = renderer.create(<Layout><div>child</div></Layout>)

  it('should match snapshot', () => {
    expect(item.toJSON()).toMatchSnapshot()
  })
})
