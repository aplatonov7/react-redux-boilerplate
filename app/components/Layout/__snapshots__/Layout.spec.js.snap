exports[`Layout component should match snapshot 1`] = `
<div
  className="root">
  <Navigation />
  <div>
    child
  </div>
</div>
`;
